mergify_engine/queue/merge_train.py

```diff
@@ -1439,6 +1422,14 @@ async def get_queue_rules(self) -> typing.Optional["rules.QueueRules"]:
 
         return mergify_config["queue_rules"]
 
+    async def get_queue_rule(self, queue_name: "rules.QueueName") -> "rules.QueueRule":
+        rules = await self.get_queue_rules()
+
+        if not rules or queue_name not in [rule.name for rule in rules]:
+            raise RuntimeError("Rules are missing from configuration")
```

Comment:
Can't we add more details on this error ? Like the queue name and/or the rule name ?